docs/setup-cluster/slurm/slurm-requirements.rst

```diff
@@ -228,6 +228,94 @@ interacts with PBS, we recommend the following steps:
    define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
    your experiment configuration to indicate how many GPU slots are intended for Determined to use.
 
+-  Configure PBS to manage GPU resources.
+
+   Determined works best when allocating GPUs. By default, Determined selects compute nodes with
+   GPUs using the option ``-select={slots_per_trial}:ngpus=1``. If PBS cannot be configured to
+   identify GPUs in this manner, specify the :ref:`pbs section <cluster-configuration-slurm>`
+   ``gres_supported`` option to ``false`` when configuring Determined, and it will then be the
+   user's responsibility to ensure that GPUs will be available on nodes selected for the job using
+   other configurations such as targeting a specific resource pool with only GPU nodes, or
+   specifying a PBS constraint in the experiment configuration.
+
+   PBS should be configured to provide the environment variable ``CUDA_VISIBLE_DEVICES``
+   (``ROCR_VISIBLE_DEVICES`` for ROCm) using a PBS cgroup hook as described in the PBS
+   Administrator's Guide. If PBS is not configured to set ``CUDA_VISIBLE_DEVICES``, Determined will
+   utilize a single GPU on each node. To fully utilize multiple GPUs, you must either manually
+   define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
+   your experiment configuration to indicate how many GPU slots are intended for Determined to use.
+
+-  Ensure ``ngpus`` resource is defined with appropriate values.
+
+   It is necessary to define the ``ngpus`` resource value for every node on the cluster for
+   DeterminedAI to work successfully. Additionally, the resource should also have appropriate flags
+   for PBS to process the value properly while scheduling jobs. Use the following command to check
+   if ``ngpus`` resource is defined with appropriate flags.
+
+   .. code:: bash
+
+      [~]$ qmgr -c "list resource ngpus"
+      Resource ngpus
+          type = long
+          flag = hn
+
+   The above output indicates that ngpus is defined as a type ``long`` resource with flags ``h``
+   (indicating this is a host level resource) and ``n`` (indicating this is a consumable resource).
+   If the nodes on your hpc cluster do not provide the same output, use the following commands to
+   create and set the resource ``ngpus`` for every node.
+
+   .. code:: bash
+
+      [~]$ qmgr -c "create resource ngpus type=long, flag=hn" # create the ngpus resource
+      [~]$ qmgr -c "set node <nodename> ngpus=<number of GPUs>" # set the value for ngpus
+
+   If you are using virtual nodes (vnodes), ensure ``ngpus`` value is only set on the vnodes and not
+   the actual parent node. Below are the commands and sample output to ensure the same.
+
```

Comment:
If you use virtual nodes (vnodes), make sure the ``ngpus`` value is set only on the vnodes, not
   the parent node. Below are the commands and sample output to ensure this: